server.js

```diff
@@ -219,8 +219,19 @@ if (useAutoStoreData === 'true') {
 }
 
 // Clear all data in session if you open /prototype-admin/clear-data
+// Reset all data in session with what's in the sessionDataDefaults file.
 app.post('/prototype-admin/clear-data', function (req, res) {
   req.session.data = {}
```

Comment:
We can remove this because we'll set it later anyway – even if we fail to load the JSON then `sessionDataDefaults` will just be `{}` and we'll still set it anyway.

```suggestion
```